data/anasouardini.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "Anas Ouardini",
+  "bio": "Web developer, mainly javaScript.",
+  "tags": [
+    "Javascript",
+    "Typescript",
+    "ReactJs",
+    "nodeJs",
+    "ExpressJs",
+    "MySql"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/segfaulty1"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/anasouardini"
+    },
+    {
+      "icon": "FaLinkedIn",
```

You're probably using an invalid icon here. It should be `FaLinkedin`.

Rest looks good to me! :)

```suggestion
      "icon": "FaLinkedin",
```